packages/api/src/auth/auth.module.ts

```diff
@@ -0,0 +1,29 @@
+import { Module } from '@nestjs/common';
+import { JwtModule } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
+
+import { ConfigService } from '../shared/config.service';
+import { SharedModule } from '../shared/shared.module';
+import { UsersModule } from '../users/users.module';
+import { UsersService } from '../users/users.service';
+import { AuthController } from './auth.controller';
+import { AuthService } from './auth.service';
+import { GithubClient } from './strategies/github.client';
+
+@Module({
+  imports: [
+    PassportModule.register({ defaultStrategy: 'jwt' }),
```

Comment:
Is JWT strategy related to GitHub? Or it's totally different strategy?

Reply:
Yes, it's separate strategy

Reply:
So I don't see this separation in the modules. 
Let's ask yourself: Can JWT authorization exists without GitHub? And vice versa? 

Answer is YES ((maybe not, and I'm missing something?). So you should have two different modules - one for GitHub and one for internal / JWT strategy.